projects/driver_controls/src/fsm/power_fsm.c

```diff
@@ -5,8 +5,14 @@
 //      to transition from the off to the charging state, and vice versa
 //    - Pressing the power button while the mechanical brake is held down will cause the car
 //      to transition between the off and the on state.
+// Note that the additional "brake" states are to ensure that braking behavior is consistent
+// i.e. If the mechanical brake is held during a fault, it does not need to be released and
+// reapplied when pressing the power button to enter drive. If these states did not exist, the car
+// would enter charging instead as the mechanical brake state is lost.
+// The alternative would be to use guarded transitions and expose the mechanical brake state.
```

Comment:
I'll probably just use a guarded transition on the mechanical brake module once it's implemented.

Reply:
SGTM